filebeat/etc/filebeat.yml

```diff
@@ -30,6 +30,14 @@ filebeat:
       # * stdin: Reads the standard in
       input_type: log
 
+      # Exclude lines. A list of regular expressions to match. It drops the lines that are
+      # matching any regular expression from the list. By default, no lines are dropped. 
+      # exclude_lines: ["^DBG"]
```

Comment:
One tiny detail: We always have not space after # for the actual config. This make it possible to differentiate it from the comment and make uncommenting easier. Same below.